P5/Source/Specs/gram.xml

```diff
@@ -152,23 +168,22 @@ $Id$
         </valItem>
       </valList>
       <remarks versionDate="2010-07-07" xml:lang="en">
-        <p>A much fuller list of values for the <att>type</att>
-        attribute may be generated from the data category registry
-        accessible from <ptr target="http://www.isocat.org/"/> and
-        maintained by ISO technical committee 37. <!--TC 37/SC 3/WD 12
+        <p>A much fuller list of values for the <att>type</att> attribute may be generated from the
+          data category registry accessible from <ptr target="http://www.isocat.org/"/> and
```

Comment:
change of the isocat reference is needed

Reply:
embarrassing... :-) Sure, going to fix that now, thanks :-)

Reply:
Actually, this one is a tougher issue. I'll leave some vague statement here, or else I'd have to propose more than I was asked to (and there's no time, in this round, I'm afraid).

Reply:
because there is no _reliable_ source to recommend nowadays, and the only one I can think of is the Lex0 suggestion at https://dariah-eric.github.io/lexicalresources/pages/TEILex0/spec.html#TEI.gram (unfortunately, it's hard-coded)

Reply:
So I got a bit stuck here (well, for 40 minutes, it seems), trying to find a better solution than just deleting the `<remarks>`.

Reply:
I'm resolving this tentatively -- there is still work to do (_systemic_ work!) but I suggest that (unless you have a better solution) we leave this with the bland content that I have replaced the reference to ISOCat with.